TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -50,42 +49,73 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
         }
 
         view.findViewById<FloatingActionButton>(R.id.add_fab).setOnClickListener {
-            addItem()
+            DataProvider.createNote()
+
+            // Set selected item to newly created note (first element in list)
+            setSelectedItem(0)
+            startNoteFragment(selectedItemPosition)
         }
 
         return view
     }
 
-    private fun addItem() { }
-
     private fun setSelectedItem(position: Int) {
```

Comment:
can `position` be returned here? then we don't have to set global variable `selectedItemPosiiton`